js/actions/sources.js

```diff
@@ -3,13 +3,16 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 "use strict";
 
-const constants = require('../constants');
 const promise = require('devtools/sham/promise');
 const Services = require('devtools/sham/services');
 const { dumpn } = require("devtools/shared/DevToolsUtils");
 const { PROMISE, HISTOGRAM_ID } = require('devtools/client/shared/redux/middleware/promise');
-const { getSource, getSourceText } = require('../queries');
 const { Task } = require('devtools/sham/task');
+const SourceUtils = require('devtools/client/shared/source-utils');
+
+const { getSource, getSourceText } = require('../queries');
+const constants = require('../constants');
+const Prefs = require('../prefs');
```

Comment:
Made a minor style decision to load ff-devtools-lib modules before debugger.html modules. Feel free to push back